SalesforceFoundation/HEDAP
# Feature/xinnanli/blank addr bug

# Critical Changes

# Changes
We no longer create empty address records when the Contact doesn't contain Primary Address information.

# Issues Closed
Fixes #588
# New Metadata

# Deleted Metadata

# Testing Notes
1. Go to an existing contact
2. Select a primary address type without entering other Address details
3. Click save and make sure no empty address is created
